src/config.json

```diff
@@ -51551,7 +51551,6 @@
     "orblterr.online",
     "orditerr.online",
     "paincakeswap.com",
-    "pancakeswap.com",
```

Comment:
The DNS records of `pancakeswap.com` and `pancakeswap.finance` appear to be the same, which is the reason for the removal here. (the .com also redirects to the correct site)